src/views/List.jsx

```diff
@@ -7,11 +7,16 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{
+					/**
+					 * TODO: write some JavaScript that renders the `data` array
+					 * using the `ListItem` component that's imported at the top
+					 * of this file.
+					 */
+					data.map((item) => (
+						<ListItem name={item.name} key={item.id} />
```

Comment:
Code looks good! You can also choose to destructure the item object to pass the attributes without dot notation, if you wish.

`{data.map(({ name, id }) => (
					<ListItem key={id} name={name} />
				))}`